README.md

```diff
@@ -1068,7 +1068,7 @@ Terminology differs somewhat between various database systems; please refer to t
 
 ## Limitations
 
-SQL 2017 detection support has been added. This support is limited to functionality already present for other versions. No new SQL 2017 specific functionality has been added in this release.
+SQL 2017 and 2019 detection support has been added. This support is limited to functionality already present for other versions. No new SQL 2017 specific functionality has been added in this release.
```

Comment:
Should we update the notes about polybase for 2017 and 2019?

Reply:
Fixed